vuepress/docs/v6.3.2/tutorials/customize-the-platform/pda-tutorial.md

```diff
@@ -1,344 +1,233 @@
-
-### PAM Implementation
-
-The PAM integration renders task variables in the task details widget.
+### Installation Steps
 
-## Task Comments
+1. Apply the definitions for the bundle that comprises the PDA plugin. You'll need to adjust the `-n entando` option in each command to match your namespace or project.
+```
+ent bundler from-git -r https://github.com/entando-samples/entando-process-driven-plugin-bundle.git -d | ent kubectl apply -n entando -f -
+```
 
-The task comments MFE gives the user the ability to view notes attached
-to a given task and to add new notes to a task.
+2. Log into your App Builder instance.
 
-### PAM Implementation
+3. Select `Repository` from the menu on the left. Your bundles will be visible in the repository as shown in the screenshot below.
 
-The PAM implementation reads and publishes notes to the comments
-endpoint
+4. Select `Install` to install the bundle. An installation can take several minutes while the application downloads the Linux images for the microservices and installs the related assets.
 
-## Task Forms
+5. Setup permissions to configure the service by following the [Initial Deploy instructions](https://github.com/entando/entando-process-driven-plugin#initial-deploy). Note that Step 1 in this sequence applies to Entando 6.3.0 only.
 
-The task form implementation renders a form specific to a task and gives
-the user the ability to complete that form. The form implementation is a
-wrapper around a JSON schema that describes the layout, style and
-content of the form. It is up to the backend implementation to transform
-to the schema and default ux layout needed to render the form. See the
-technical documentation below for more on the JSON schema based
-implementation.
-
-### PAM Implementation
+6. Access the PDA plugn via one of the following options:
 
-The PAM implementation of forms depends on the presence of a form
-definition being present on the PAM task. The Entando PAM engine
-implementation transforms from the PAM format to the JSON schema to
-render the form. And it transforms the API format back to the PAM format
-based on the form definition in PAM. There are some limitations on form
-customization due to the format required to return data to PAM. See the
-forms section in the technical documentation for more information.
+**Option 1** If you'd like to make the PDA plugin your default home page, go to `App Builder → Pages → Settings`. In
+   the dropdown for Home Page, select `Home / xxxx` and click `Save`.
 
-## Attachments
+You can now navigate to your plugin's home page using the home icon in the upper right of the App Builder.
 
-The attachments MFE gives the user to view documents attached to a
-task/case/process and to add new documents to a task/case/process.
 
-### PAM Implementation
+**Option 2** Alternatively, you can view the PDA plugn home page by going to `Pages → Management`, finding `xxxx` in the page tree, and clicking `View Published Page` from its actions.
```

Comment:
Typo "plugn"

Reply:
Let's point them to the actual page name, PDA Smart Inbox is probably the natural starting point. We can be specific when there's an obvious starting point.